dataprofiler/profilers/profile_builder.py

```diff
@@ -2373,7 +2393,12 @@ def _merge_null_replication_metrics(self, other: StructuredProfiler) -> Dict:
             sum_not_null = np.delete(total_row_sum, col_id) - sum_null
 
             mean_null = sum_null / null_count
-            mean_not_null = sum_not_null / true_count
+
+            mean_not_null = np.full(len(self._profile) - 1, np.nan)
+            if not true_count:
+                sum_not_null = np.full(len(self._profile) - 1, np.nan)
+            else:
+                mean_not_null = sum_not_null / true_count
```

Comment:
this how this code was previously
```
            sum_not_null = np.delete(total_row_sum, col_id) - sum_null

            mean_null = sum_null / null_count
            mean_not_null = sum_not_null / true_count
```
there is a concern that `np.full()` is getting run after `np.delete()`. Can we validate that is not materially changing anything downstream. This comes across as a larger change to the source code.